### PR TITLE
support setting uid when injecting fault

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

user can set the uid when injecting fault, it is more convenient for managing the experiment.

### CODE REVIEW

File: pkg/server/chaosd/attack.go [hunk 1]
```diff
@@ -51,7 +51,10 @@ func (s *Server) ExecuteAttack(attackType AttackType, options core.AttackConfig,
 		return
 	}
 
-	uid = uuid.New().String()
+	uid = options.GetUID()
```
Comment (fewdan) [inline pkg/server/chaosd/attack.go:54]: Is it because we can directly specify a UID?

**Review by fewdan — APPROVED** [review]
rest LGTM

File: pkg/server/chaosd/attack.go (hunk 1 above)
Comment (WangXiangUSTC) [reply pkg/server/chaosd/attack.go:54]: Yes

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Pull request merged**